uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/helpers/MessageHelpersTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_MessageHelpersTrait {
```

Comment:
Since this helper doesn't have only Message related helpers, but helpers related to all the classes inside the Mail namespace (and relatives), why don't we rename it to `MailHelpersTrait`?